_pages/saml.md

```diff
@@ -73,14 +73,22 @@ Here are values needed to configure your service provider (SP) to work with logi
 
 Consistent with the [SAML metadata specification](https://docs.oasis-open.org/security/saml/v2.0/saml-metadata-2.0-os.pdf), login.gov's metadata is available at [https://idp.int.identitysandbox.gov/api/saml/metadata2019](https://idp.int.identitysandbox.gov/api/saml/metadata2019)
 
+## Annual Certificate Rotation
+
+The Login.gov SAML certificate is valid for just over one year. Every spring, Login.gov adds new SAML endpoints with the current year that use a new signing certificate. (So `/api/saml/metadata2019` becomes `/api/saml/metadata2020`.)
+
+The certificates are issued to create an overlap period of about a month, during which all partners using SAML should migrate at their convenience to the new endpoint URLs for the current year.
+
+The 2019 certificates for idp.int.identitysandbox.gov and secure.login.gov each expire on April 1, 2020. So the transition from 2019 to 2020 endpoints should take place in February or March 2020.
+
```

Comment:
YES. This is great to finally have formally defined. I want to follow up with canonical endpoints (eg. /api/saml/metadata) which always responds with the current endpoint while maintaining the year suffixes to allow rotation to the new endpoints. Plus, recommend ingestion of metadata.